woningwaardering/stelsels/zelfstandige_woonruimten/punten_voor_de_woz_waarde/punten_voor_de_woz_waarde.py

```diff
@@ -148,132 +189,217 @@ def _corrigeer_woz_punten(
                     criterium=WoningwaarderingResultatenWoningwaarderingCriterium(
-                        naam=f"Nieuwbouw: min. {minimum_punten} punten"
+                        naam=f"Nieuwbouw: min. {minimum_woz_punten} punten"
                     ),
-                    punten=minimum_punten - huidige_punten,
+                    punten=float(minimum_woz_punten - woz_punten),
                 )
             )
             return woningwaardering_groep
 
-        correctie_punten = self._cap_punten(huidige_punten, woningwaardering_resultaat)
+        # Bereken de overige punten door de punten van alle groepen op te tellen, afgerond op 0 decimalen
+        overige_punten = utils.rond_af(
+            sum(
+                Decimal(str(groep.punten)) or Decimal("0")
+                for groep in woningwaardering_resultaat.groepen or []
+                if groep.punten
+            ),
+            0,
+        )
+
+        totaal_punten_zonder_cap = overige_punten + utils.rond_af(woz_punten, 0)
+        correctie_punten = self._cap_punten(woz_punten, overige_punten)
 
         if (
-            correctie_punten < 0.0
-            and minimum_punten == 0.0
+            correctie_punten is not None
+            and minimum_woz_punten == 0.0
             and woningwaardering_groep.woningwaarderingen is not None
         ):
-            woningwaardering_groep.woningwaarderingen.append(
-                WoningwaarderingResultatenWoningwaardering(
-                    criterium=WoningwaarderingResultatenWoningwaarderingCriterium(
-                        naam="Max. 33% van totaal"
-                    ),
-                    punten=correctie_punten,
-                )
+            totaal_punten_met_cap = totaal_punten_zonder_cap + correctie_punten
+
+            logger.debug(
+                f"Eenheid {eenheid.id}: Waardering zonder cap: {totaal_punten_zonder_cap} punten. Na toepassing van cap: {totaal_punten_met_cap} punten."
             )
-            return woningwaardering_groep
+
+            # Wanneer een woning zonder die beperking een waardering heeft van meer dan
+            # 186 punten en door deze beperking een waardering krijgt die lager is dan
+            # 187 punten, geldt een waardering van 186 punten voor de woning.
+            if totaal_punten_zonder_cap > 186 and totaal_punten_met_cap < 187:
+                logger.info(
+                    f"Eenheid {eenheid.id} wordt gewaardeerd met 186 punten totaal door de cap op de WOZ voor de stelselgroep {Woningwaarderingstelselgroep.punten_voor_de_woz_waarde.naam}"
+                )
+                correctie_punten = 186 - totaal_punten_zonder_cap
+                woningwaardering_groep.woningwaarderingen.append(
+                    WoningwaarderingResultatenWoningwaardering(
+                        criterium=WoningwaarderingResultatenWoningwaarderingCriterium(
+                            naam="Maximering WOZ-punten tot 186 punten totaal"
```

Comment:
```suggestion
                            naam="Maximaal 186 punten voor WOZ-waarde"
```
Dit zou meer overeenkomen met eerder taalgebruik omtrent maximaal aantal punten (en misschien net iets laagdrempeliger Nederlands)

Reply:
nvm, ik snap nu wat je bedoelt!